edenai_apis/apis/winstonai/winstonai_api.py

```
@@ -77,6 +77,44 @@ def image__ai_detection(
             standardized_response=standardized_response,
         )
 
+    async def image__aai_detection(
+        self, file: Optional[str] = None, file_url: Optional[str] = None, **kwargs
+    ) -> ResponseType[ImageAiDetectionDataclass]:
+        if not file_url and not file:
+            raise ProviderException("file or file_url required")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)